package.json

```diff
@@ -70,6 +70,7 @@
     "b2a": "^1.0.10",
     "bad-words": "^1.6.4",
     "body-parser": "^1.18.3",
+    "brianlovin-react-tippy": "^1.2.4",
```

Comment:
`react-tippy` is using an ancient version of `tippy.js` from 2017 and is effectively unmaintained -- I recommend switching to [`@tippy.js/react`](https://www.npmjs.com/package/@tippy.js/react) which is actively maintained! CodeSandbox is migrating to it now - see this PR: https://github.com/CompuIves/codesandbox-client/pull/1584

Reply:
Thanks for the followup on this @atomiks! Will try to upgrade.